inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sink/mysql/MySQLSinkDTO.java

```diff
@@ -57,8 +57,14 @@ public class MySQLSinkDTO {
 
         {
             put("autoDeserialize=true", "autoDeserialize=false");
+            put("autoDeserialize=yes", "autoDeserialize=false");
```

Comment:
Can we just check the key `allowUrlInLocalInfile`, not including its value, no matter if it is `=true` or `=yes`?

Reply:
Ok, it has been changed to this way